charts/sda-svc/test/release-test.sh

```diff
@@ -1,40 +1,41 @@
 #!/bin/bash
 
-if [ "${DEPLOYMENT_TOPOLOGY}" = "standalone" ]; then
-  ingest_routing_key="files";
-  accession_routing_key="files";
+if [ "${DEPLOYMENT_TOPOLOGY}" = "federated" ]; then
```

Comment:
i assume this change is cause by the issue at: https://github.com/neicnordic/sda-helm/pull/104/files#diff-9d268e106887434703f8991f912c806e529de5c79261c2e2831a997e46d440f7R84

Reply:
The original logic was backwards since `intercept -> federated`